harness/determined/pytorch/_load.py

```diff
@@ -23,7 +23,9 @@ class CheckpointLoadContext(pytorch.PyTorchTrialContext):
     Users who are relying on the legacy `entrypoint: my_model:MyTrainer` way of launching their code
```

Comment:
From the indentation, there appear to be a lot of paragraphs in this block of text above that need empty lines between them.

Some empty lines in the code example of this function wouldn't hurt either, to improve readability a bit.